Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -52,6 +53,17 @@ void __InitDWriteFontPropertiesFromName(
     return s_dwriteFactory;
 }
 
+// Private helper for converting a DWRITE_FONT_METRIC unit to a CTFont API unit
+// DWRITE_FONT_METRICS measures its metrics in 'design units'
+// CTFont APIs generally return in terms of point size
+CGFloat _CoreTextScaleMetric(CGFloat metric, CGFloat pointSizeUnitsPerEm) {
+    // design units * (pt / em) / (design units / em)
+    // = design units * (pt / em) * (em / design units)
+    // = pt * (design units / design units) * (em / em)
+    // = pt
```

Comment:
these comments don't actually apply to the math here anymore also...

---
In reply to: [80326649](https://github.com/Microsoft/WinObjC/pull/1027#discussion_r80326649) [](ancestors = 80326649)